### PR TITLE
Automated OCP-21926

### PR DESCRIPTION
**Pull request by wzheng1**
Here is test result: https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3/81811/console

### CODE REVIEW

**Comment by wzheng1** [comment]
@openshift/devexp-qe Please review, thanks!

File: features/cli/oc_registry.feature [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+  # @author wzheng@redhat.com
+  # @case_id OCP-21926
+  Scenario: Check function of oc registry command
+    Given I have a project
+    When I run the :registry_info client command with:
+      | internal | true |
+    Then the step should succeed
+    And the output should contain:
+      | image-registry.openshift-image-registry.svc:5000 |
```
Comment (pruan-rht) [inline features/cli/oc_registry.feature:11]: @wzheng1 does this output changes depending on the cluster configuration?  Like in-cluster registry?

Comment (wzheng1) [reply features/cli/oc_registry.feature:11]: No, all 4.x cluster has the same name like this.

File: lib/rules/cli/4.1.yaml [hunk 2]
```diff
@@ -1327,3 +1327,13 @@
...
+:registry_info:
+  :cmd: oc registry info
+  :options:
+    :internal: --internal=<value>
+    :quiet: --quiet=<value>
+:registry_login:
+  :cmd: oc registry login
+  :options:
+    :z: -z default
+    :skip-check: --skip-check=<value>
```
Comment (liangxia) [inline lib/rules/cli/4.1.yaml:1339]: Sort them with alphanumeric order.

Comment (wzheng1) [reply lib/rules/cli/4.1.yaml:1338]: Done

File: features/cli/oc_registry.feature (hunk 1 above)
Comment (xiuwang) [reply features/cli/oc_registry.feature:11]: @wzheng1 If this will report error when the route for default regtistry is enabled?

Comment (wzheng1) [reply features/cli/oc_registry.feature:11]: No, that's why I use internal=true option

**Pull request merged**